UnityProjects/MRTKDevTemplate/Assets/Scripts/ScrollRectCurve.cs

```diff
@@ -32,7 +32,7 @@ public class ScrollRectCurve : MonoBehaviour
 
         void Awake()
         {
-            scrollRect = scrollRect ?? gameObject.GetComponentInParent<ScrollRect>(true);
+            scrollRect = (scrollRect != null) ? scrollRect : gameObject.GetComponentInParent<ScrollRect>(true);
```

Comment:
Does assigning `scrollRect` to itself _do_ anything, or could this be
```suggestion
            if (scrollRect == null)
            {
                scrollRect = gameObject.GetComponentInParent<ScrollRect>(true);
            }
```

Reply:
Ope, I was on autopilot remove-all-null-coalescers mode...